### ISSUE TITLE
Split up one list into easier digestible subpages

### ISSUE THREAD

**Issue by therealprof** [rust-embedded/awesome-embedded-rust#20]
Even though the new `awesome-embedded-rust` project just started, it's already getting a bit unwieldy to read and edit so I'd like to split it up in more manageable chunks so for starters I'd like to take the crate heavy sections (i.e. Device crates, HAL implementations, BSP crates and Drivers) and put them each into their own `.md` file, move the vendor entries from the subsections to where the individual crates are now and make them into direct links to their new location (where available, drivers should also be categorized at some point).

Once that is done I'd like to follow up with enriching the content quite a bit and also to bring back the wish list/idea pool which @japaric started in rust-lang-nursery/embedded-wg/issues/39.

Thoughts or comments?

**Comment by berkus** [comment]
How does splitting into multiple page goes with the awesome.re guidelines? I thought it's supposed to be one large list of everything.

**Comment by therealprof** [comment]
Huh? https://github.com/sindresorhus/awesome/blob/master/awesome.md doesn't say anything about being in one page and I've also seen some lists that had sub-pages.

Anyway, in the end the list should be a useful resource to interested parties and not follow some random scheme for the sake of following some random scheme.

**Comment by berkus** [comment]
Sure thing.

**Comment by RandomInsano** [comment]
Seems good to me! I’d also like the tablify some sections (like the WIP list), but I’m not sure when I’ll be able to get around to it.
